x-pack/plugins/apm/server/lib/helpers/aggregated_transactions/get_fallback_to_transactions.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { getSearchAggregatedTransactions } from '.';
+import { SearchAggregatedTransactionSetting } from '../../../../common/aggregated_transactions';
+import { Setup, SetupTimeRange } from '../setup_request';
+
+export async function getFallbackToTransactions({
+  setup: { config, start, end, apmEventClient },
+  kuery,
+}: {
+  setup: Setup & Partial<SetupTimeRange>;
+  kuery?: string;
+}): Promise<boolean> {
```

Comment:
I've previously been against setting explicit return types that can inferred but I think it makes a lot of sense for simple types that are not expected to change, like in this case.
It also makes the code much easier to read in a code review.